skorch/tests/test_helper.py

```diff
@@ -474,3 +482,230 @@ def test_pickle(self, predefined_split, data):
 
         # does not raise
         pickle.dumps(train_split)
+
+
+class TestDataFrameTransformer:
+    @pytest.fixture
+    def transformer_cls(self):
+        from skorch.helper import DataFrameTransformer
+        return DataFrameTransformer
+
+    @pytest.mark.skipif(not pandas_installed, reason='pandas is not installed')
+    @pytest.fixture
+    def df(self):
+        """DataFrame containing float, int, category types"""
+        import pandas as pd
+
+        df = pd.DataFrame({
+            'col_floats': [0.1, 0.2, 0.3],
+            'col_ints': [11, 11, 10],
+            'col_cats': ['a', 'b', 'a'],
+        })
```

Comment:
I am a little surprised some of the test pass without this in python 3.5:

```suggestion
        }, columns=['col_floats', 'col_ints', 'col_cats'])
```

Reply:
Why indicate the column names a second time?